app/src/androidTest/java/com/github/ybecker/epforuml/AnswerAdapterTest.kt

```diff
@@ -32,35 +37,41 @@ class AnswerAdapterTest {
     private var question3 = db.getQuestionById("question3")
 
 
+
+
     @Before
     fun setup() {
         DatabaseManager.useMockDatabase()
-        db.getUserById("user1").thenAccept{
+        db.getUserById("answerAdapterTestUser").thenAccept{
```

Comment:
This user "answerAdapterTestUser" is not in the database, it mean that here you just add a question to a null user. The question will go in the database anyway, but consider adding the answerAdapterTestUser in the DB or deleted this getById call.